docs/worker-node/install-singularity.md

```diff
@@ -133,17 +133,20 @@ RedHat when they are discovered.
 
 ### Validating singularity ###
 
-After singularity is installed, as an ordinary user run the following
-command to verify it:
-
 ```console
-user@host $ singularity exec -c --ipc --pid -H $HOME:/srv /cvmfs/singularity.opensciencegrid.org/opensciencegrid/osgvo:el6 \
+user@host $ singularity exec -c --ipc --pid \
+                --home $PWD:/srv \
+                --pwd /srv \
+                /cvmfs/singularity.opensciencegrid.org/opensciencegrid/osgvo:el6 \
                 ps -ef
```

Comment:
This is the command that John was using to test Singularity 2.4.6, can we reproduce all of the options?

```
/usr/bin/singularity exec --home /var/lib/condor/execute/dir_27131/glide_wKlQBG/tmp.JrORzc7o07:/srv --bind /cvmfs --pwd /srv --scratch /var/tmp --scratch /tmp --contain --ipc --pid /cvmfs/singularity.opensciencegrid.org/opensciencegrid/osgvo-el6:latest echo "Hello World"
```